src/resources/dev-envs/clion.md

```diff
@@ -0,0 +1,80 @@
+# CLion
+
+CLion is a commercial IDE from JetBrains. Altough it is expensive for regular users, it is free for university students. <br>
```

Comment:
Fix spelling mistakes. This is also not entirely correct here either. There are many other uses that provide free and discounted CLion licenses.